ci-operator/step-registry/mirror-images/icsp-olm-operators/mirror-images-icsp-olm-operators-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+mirror_auths="${SHARED_DIR}/mirror_auths"
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy:"
+        cat "${SHARED_DIR}/proxy-conf.sh"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "no proxy setting."
+    fi
+}
+
+# private mirror registry host
+# <public_dns>:<port>
+MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
```

Comment:
If we want to use proxy-registry, there are two proxy registry ${MIRROR_REGISTRY_HOST1}:6000, ${MIRROR_REGISTRY_HOST2}:6002.   And we have to write their credential the cluster.

Reply:
> And we have to write their credential the cluster.

Line 31 is the auth.